src/nvim/os/win_defs.h

```diff
@@ -3,6 +3,8 @@
 
 #include <windows.h>
 
+#include <uv.h>
```

Comment:
Hopefully we can avoid this, we want to encapsulate libuv in the `os/` layer. Modules that import `os_defs.h` should not get libuv as well.